### DISCUSSION TITLE
Não consigo enviar o documento para cadasto no GitHub Education

### DISCUSSION THREAD

**Discussion by nutricionistaafonso** [frontendbr/forum#2429] [Perguntas e Respostas]
Sempre que tento enviar o aquivo dá erro, já tentei atestado de matricula e cartão de estudante, sempre dá erro

**Comment by RuanAragao** [comment]
Dar erro no anexo ou o tipo de comprovante é rejeitado pela avaliação?

**Comment by nutricionistaafonso** [comment]
era pra reconhecer o anexo, mas consegui depois
